src/Symfony/Component/Notifier/Bridge/Mercure/Tests/MercureTransportTest.php

```diff
@@ -114,7 +114,7 @@ public function testSendWithMercureOptions()
     {
         $hub = new MockHub('https://foo.com/.well-known/mercure', new StaticTokenProvider('foo'), function (Update $update): string {
             $this->assertSame(['/topic/1', '/topic/2'], $update->getTopics());
-            $this->assertSame('{"@context":"https:\/\/www.w3.org\/ns\/activitystreams","type":"Announce","summary":"subject"}', $update->getData());
+            $this->assertSame('{"@context":"https:\/\/www.w3.org\/ns\/activitystreams","type":"Announce","summary":"subject","mediaType":"application\/json","content":null}', $update->getData());
```

Comment:
I think we should add a test where content is not `null` as this is the goal off this PR.